app/db/seeder.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'bcrypt'
+require 'faker'
+require 'sequel'
+require 'literate_randomizer'
+
+DB = Sequel.connect('sqlite://db/development.db')
+DB[:users].truncate
+DB[:messages].truncate
+DB[:followers].truncate
+users = DB[:users]
+messages = DB[:messages]
+followers = DB[:followers]
+
+10.times do
+  users.insert(
+    email: Faker::Internet.email,
+    username: "#{Faker::Name.first_name}_#{Time.now.to_i}",
+    password: BCrypt::Password.create('secret')
+  )
+end
+
+10.times do
+  followers.insert(
+    who_id: users.all.sample[:user_id],
+    whom_id: users.all.sample[:user_id]
+  )
+end
```

Comment:
It looks to me like the seeding allows for a user to follow himself?

Reply:
Yes, I actually what would happen in the frontend.